app/graphql/types/volume.rb

```diff
@@ -0,0 +1,41 @@
+class Types::Volume < Types::BaseObject
+  description 'A manga volume which can contain multiple chapters.'
+
+  field :id, ID, null: false
+
+  field :titles, Types::TitlesList,
+    null: false,
+    description: 'The titles for this chapter in various locales'
+
+  def titles
+    {
+      localized: object.titles,
+      canonical: object.canonical_title
+    }
+  end
+
+  field :number, Integer,
+    null: false,
+    description: 'The number of pages in this chapter.'
```

Comment:
Is this the chapter count or page count?

Reply:
actually that is just wrong, that is the volume number.